### PR TITLE
WAZO-3429 update to asterisk 21

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

File: debian/source/options [hunk 1]
```diff
@@ -1 +1 @@
-extend-diff-ignore = "(^|/)(\.dockerignore|AUTHORS|Dockerfile|README.md|CHANGELOG.md|LICENSE|integration_tests.*|contribs.*|zuul.yaml|tox.ini|Jenkinsfile)$"
+extend-diff-ignore = "(^|/)(\.dockerignore|AUTHORS|Dockerfile|README.md|CHANGELOG.md|LICENSE|integration_tests.*|contribs.*|zuul.yaml|tox.ini|Jenkinsfile|.lastclean)$"
```
Comment (fblackburn1) [inline debian/source/options:1]: This file always exist on asterisk project. Why it creates issue now?

**Review by fblackburn1 — COMMENTED** [review]
In https://wazo-dev.atlassian.net/browse/WAZO-3429 there is a note to check if we should add an option `-with-libjwt-bundled`. Should we add it or it's already OK by default?

**Comment by sduthil** [comment]
> In https://wazo-dev.atlassian.net/browse/WAZO-3429 there is a note to check if we should add an option `-with-libjwt-bundled`. Should we add it or it's already OK by default?

I was intending to test this after getting the first build :)

File: debian/source/options (hunk 1 above)
Comment (fblackburn1) [reply debian/source/options:1]: My bad, I didn't expand the commit message :pray:

**Comment by sduthil** [comment]
> > In https://wazo-dev.atlassian.net/browse/WAZO-3429 there is a note to check if we should add an option `-with-libjwt-bundled`. Should we add it or it's already OK by default?
> 
> I was intending to test this after getting the first build :)

I just realized that this option will be necessary for all upcoming version of Asterisk, not only Asterisk 21. And the version requiring this option has not been released by the Asterisk team yet.

**Pull request merged**